corehq/apps/hqwebapp/static/hqwebapp/scss/commcarehq/_select2.scss

```diff
@@ -73,3 +73,86 @@
 .select2-selection__placeholder {
   color: $gray-base !important;
 }
+
+.select2-container--default .select2-selection--single,
+.select2-container--default .select2-selection--multiple {
+  border-color: $border-color !important;
+  height: 32px !important;
```

Comment:
Is this to align select2 height with the height of text inputs?

Reply:
yes

Reply:
the text inputs in bootstrap 5 use a y-padding value, but select2 goes by a specific height

Reply:
awesome